.circleci/get_pinned_chrome_version.sh

```diff
@@ -46,7 +47,18 @@ echo "$(GREEN "Chrome version history URL is") $(CYAN "${CHROME_VERSION_HISTORY_
 # Determine the Chrome version
 # See https://developer.chrome.com/docs/versionhistory/guide
 echo "$(GREEN "Determining Chrome version...")"
-CHROME_VERSION="$(curl -sS ${CHROME_VERSION_HISTORY_URL} | jq -r ".versions[]|.version" | grep -m 1 "${CHROME_MAJOR_VERSION}\.")"
+for attempt in {2..0}
```

Comment:
I wonder if we can simplify things by simply changing the `curl` command to `curl --retry 3`. With exponential backoff built in to `--retry`, this should virtually guarantee a result if the version history backend is indeed available. And if it isn't, it's okay for the job to fail, although it will be significantly more rare. WDYT?

Reply:
Ha, this is so much better! The common error is `429` which is immediately retriable with `curl --retry`, so I reverted the change and just added `--retry 3` here. It also does exponential back-off on its own and respects `Retry-After` response headers